Detectors/MUON/MCH/Simulation/include/MCHSimulation/Digitizer.h

```diff
@@ -40,6 +40,8 @@ class Digitizer
   void process(const std::vector<Hit> hits, std::vector<Digit>& digits);
   void provideMC(o2::dataformats::MCTruthContainer<o2::MCCompLabel>& mcContainer);
   void mergeDigits(const std::vector<Digit> digits, const std::vector<o2::MCCompLabel> trackLabels);
+  //external pile-up adding up
+  void mergeDigits(std::vector<Digit>& digits, const o2::dataformats::MCTruthContainer<o2::MCCompLabel> mcContainer);
```

Comment:
any reason why the mcContainer is passed by value in this one ?

Reply:
@aphecetche : no, I could also do by reference and add inside this, the provideMC instead of writing it afterwards in MCHDigitizerSpec.cxx.

Reply:
This is now changed.